### PR TITLE
feat: taosws python lib support pep-249

### PR DESCRIPTION
**Pull request by zhaoyanggh**

### CODE REVIEW

**Review by zitsen — APPROVED** [review]
LGTM

File: taos-ws-py/examples/pandas-read-sql.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import pandas
+from sqlalchemy import create_engine
+import taosws
+
+engine = taosws.connect("taos://localhost:6041")
```
Comment (sangshuduo) [inline taos-ws-py/examples/pandas-read-sql.py:5]: current pattern is native use taos://...:6030, REST use taosrest://...:6041, should we use taosws://...:6041 to make user explicitly know using websocket?

Comment (sangshuduo) [reply taos-ws-py/examples/pandas-read-sql.py:5]: I guess here we should demo how create_engine work with py connector?

Comment (zitsen) [reply taos-ws-py/examples/pandas-read-sql.py:5]: > current pattern is native use taos://...:6030, REST use taosrest://...:6041, should we use taosws://...:6041 to make user explicitly know using websocket?

Aggreed.



> I guess here we should demo how create_engine work with py connector?

Codes ready. Need to publish new taospy version.

**Pull request merged**